htdocs/class/xoopsform/renderer/XoopsFormRendererTailwind.php

```
@@ -0,0 +1,821 @@
+     * @param XoopsFormTextDateSelect $element form element
+     *
+     * @return string rendered form element
+     */
+    public function renderFormTextDateSelect(XoopsFormTextDateSelect $element)
+    {
+        static $included = false;
+        if (file_exists(XOOPS_ROOT_PATH . '/language/' . $GLOBALS['xoopsConfig']['language'] . '/calendar.php')) {
+            include_once XOOPS_ROOT_PATH . '/language/' . $GLOBALS['xoopsConfig']['language'] . '/calendar.php';
+        } else {
+            include_once XOOPS_ROOT_PATH . '/language/english/calendar.php';
+        }
```

PHPMD (2.15.0)

[error] The method renderFormTextDateSelect uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)